torch/_inductor/pattern_matcher.py

```diff
@@ -608,6 +608,279 @@ def addmm(match, mat1, mat2, inp):
+        CallFunction(mkldnn._convolution_pointwise.default, *_conv_args, _users=4)
+    ]
+
+    def gelu_fusion_1(computation_call):
+        return CallFunction(
+            aten.mul,
+            CallFunction(aten.mul, computation_call, 0.5),
+            CallFunction(
+                aten.add,
+                CallFunction(
+                    aten.erf,
+                    CallFunction(aten.mul, computation_call, 0.7071067811865476),
+                ),
+                1,
+            ),
+        )
+
+    def gelu_fusion_2(computation_call):
+        return CallFunction(
+            aten.mul,
+            CallFunction(aten.mul, computation_call, 0.5),
+            CallFunction(
+                aten.add,
+                CallFunction(
+                    aten.tanh,
+                    CallFunction(
+                        aten.mul,
+                        CallFunction(
+                            aten.add,
+                            computation_call,
+                            CallFunction(
+                                aten.mul,
+                                CallFunction(
+                                    aten.mul,
+                                    CallFunction(
+                                        aten.mul, computation_call, computation_call
+                                    ),
+                                    computation_call,
+                                ),
+                                0.044715,
+                            ),
+                        ),
+                        0.7978845608028654,
+                    ),
+                ),
+                1,
+            ),
+        )
+
+    def hardswish_fusion(computation_call):
+        return CallFunction(
+            aten.div,
+            CallFunction(
+                aten.mul,
+                computation_call,
+                CallFunction(
+                    aten.clamp_max,
+                    CallFunction(
+                        aten.clamp_min, CallFunction(aten.add, computation_call, 3), 0
+                    ),
+                    6,
+                ),
+            ),
+            6,
+        )
+
+    def silu_fusion(computation_call):
+        return CallFunction(
+            aten.mul, computation_call, CallFunction(aten.sigmoid, computation_call)
+        )
+
+    def hardsigmoid_fusion(computation_call):
+        return CallFunction(
+            aten.div,
+            CallFunction(
+                aten.clamp_max,
+                CallFunction(
+                    aten.clamp_min, CallFunction(aten.add, computation_call, 3), 0
+                ),
+                6,
+            ),
+            6,
+        )
+
+    def relu_fusion(computation_call):
+        return CallFunction(aten.relu, computation_call)
+
+    def sigmoid_fusion(computation_call):
+        return CallFunction(aten.sigmoid, computation_call)
+
+    def tanh_fusion(computation_call):
+        return CallFunction(aten.tanh, computation_call)
```

Comment:
You can match multiple functions in a single pattern like:
```suggestion
    def combined_fusion(computation_call):
        return CallFunction([aten.relu, aten.sigmoid, aten.tanh], computation_call)
```

Then figure out which one you hit with the `match` arg to the lowering.

This should allow you to have fewer patterns throughout the stack.

Reply:
yes, it is simplified.